src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -10,7 +10,23 @@
      * <p>Пример: если дана строка "data 48 call 9 read13 blank0a", то в массиве должны оказаться
      * числа 92, 18, 26 и 0.</p>
      */
-    public int[] getAllNumbers(String text) {
-        return null;
+    public static int[] getAllNumbers(String text) {
+        text = text.replaceAll("[^\\d-]", " ");
+        String[] numbers = text.split(" ");
+        int count = 0;
+        for (String number : numbers) {
+            if (number.length() != 0) {
+                ++count;
+            }
+        }
+        int[] result = new int[count];
+        count = 0;
```

Comment:
all these lines are redundant if we find a way do improve our split so that `numbers` would not contain empty strings